playbooks/ci_latency_tests.yaml

```diff
@@ -44,3 +44,38 @@
         -d {{ sv_interface }} \
         -f "/tmp/latency_tests/ts_{{ inventory_hostname }}.txt"
       become: yes
+
+- name: Launch pcap sending
+  hosts: publisher_machines
+  become: true
+  tasks:
+    - name: "Create pcaps directory if it doesn't exist"
+      file:
+        path: "/tmp/latency_tests/pcaps"
+        state: directory
+
+    - name: "Copy pcap file on publisher"
+      copy:
+        src: "{{ pcap_path }}/{{ pcap_file }}"
+        dest: "/tmp/latency_tests/pcaps"
+
+    - name: "Send pcap from publisher"
+      command: "bittwist -i {{ sv_interface }} \
+              /tmp/latency_tests/pcaps/{{ pcap_file }}"
+      async: 300
+      poll: 0
+      register: cmd
+
+    - name: Wait for bittwist to finish
+      async_status:
+        jid: "{{ cmd.ansible_job_id }}"
+      register: cmd_job_status
+      until: cmd_job_status.finished
+      retries: 60
+      delay: 5
```

Comment:
I don't think you need that, just launch the bittwist command, ansible will wait for it to end.